internal/sourcecode/scan.go

```diff
@@ -12,7 +13,7 @@ import (
 	"github.com/superfly/flyctl/helpers"
 )
 
-//go:embed templates/**
+//go:embed templates/** templates/**/.dockerignore
```

Comment:
the first `templates/**` should recursively glob everything, so the .dockerignore pattern might not be needed.